docs/start_3d.md

```diff
@@ -115,6 +116,10 @@ board_marker_length = 3 # mm
 
 # If charuco or checkerboard, square side length
 board_square_side_length = 4 # mm
+
```

Comment:
As above, this should go in its own section rather than with the "Calibration marker configuration"